pkg/artifactory/resource_artifactory_remote_vcs_repository.go

```diff
@@ -0,0 +1,56 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+type VcsRemoteRepo struct {
+	RemoteRepositoryBaseParams
+	VcsGitProvider     string `json:"vcsGitProvider"`
```

Comment:
You can use RemoteRepositoryVcsParams from repositories.go

Reply:
yes, using it now